jupyterlab/stage5-ro.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Set up default user directory layout
+set -e
+for i in notebooks WORK DATA idleculler ; do \
+   mkdir -p /etc/skel/${i} ; \
+done
+
+# "lsst" is a real GitHub organization, so rename the local user/group.
```

Comment:
I'm not sure how this works, but is there any reason not to make the original user name and group something other than `lsst` to begin with rather than having to move things around post-facto?

Reply:
No, that comes from the upstream build.  I mean, I guess we could petition the stack container maintainers to change it, but it's not our choice to make.